web/frontend/src/api/sensors.ts

```diff
@@ -1,20 +1,7 @@
-import { sensorsApi, Client } from '@viamrobotics/sdk';
+import { Client, sensorsApi } from '@viamrobotics/sdk';
 
 export const getSensors = async (robotClient: Client, name: string) => {
-  const request = new sensorsApi.GetSensorsRequest();
-  request.setName(name);
-
-  const response = await new Promise<sensorsApi.GetSensorsResponse | null>(
-    (resolve, reject) => {
-      robotClient.sensorsService.getSensors(request, (error, res) => {
-        if (error) {
-          reject(error);
-        } else {
-          resolve(res);
-        }
-      });
-    }
-  );
-
-  return response?.toObject().sensorNamesList ?? [];
+  const request = new sensorsApi.GetSensorsRequest({ name });
+  const resp = await robotClient.sensorsService.getSensors(request);
```

Comment:
Why not create `SensorsClient` like the other wrappers?

Reply:
because the TS SDK never had one

Reply:
note there's a SensorClient but not a SensorsClient

Reply:
not a problem then!